bin/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# syntax=docker/dockerfile:1
+# Built by script called build-dataloader.sh
+# Run with the help of scripts in bin/ such as load-data-container.sh
+
+# Dase image for the Docker image
+FROM alpine
+
+# Install the PostgreSQL client
+RUN apk add && apk add postgresql-client
+
+# Copy the entry point script to the root directory
+COPY bin/docker-entrypoint.sh /entrypoint.sh
```

Comment:
have you tested that it builds? because this relative reference is not updated.

any changes should come with testing, manual or automated